man/bootup.xml

```diff
@@ -179,6 +179,12 @@
     identical to the system manager bootup (see above) until it
     reaches <filename>basic.target</filename>. From there, systemd
     approaches the special target <filename>initrd.target</filename>.
+    When using the resume kernel parameter,
```

Comment:
Note that initrd generator may include resume information in initrd itself, so this is not really limited to kernel command line.